admin/src/containers/View/utils/parsers.js

```diff
@@ -85,6 +89,14 @@ const linkRelations = (item, config) => {
 
   const relatedItem = isArray(related) ? first(related) : related;
   const relatedId = isString(related) && !validateUUID(related) ? parseInt(related, 10) : related; 
+  const relationNotChanged = relatedRef && relatedItem ? relatedRef.id === relatedItem : false;
+
+  if (relationNotChanged) {
+    return {
```

Comment:
I am just wondering do you need to create new object (by spread), when relation is not changed maybe we could return just not Changed item `return item;`